eng/source-build-release.yml

```diff
@@ -29,10 +29,12 @@ parameters:
     default: v6.0.XYY-source-build
 
 variables:
-  repoId: ''
-  discussionCategoryId: ''
-  announcementOrg: 'dotnet'
-  announcementRepo: 'source-build'
+- group: DotNet-Source-Build-Bot-Secrets
+- name: announcementOrg
+  value: 'lbussell'
```

Comment:
Looks like you may have accidentally checked in the testing values to the Org and Repo

Reply:
Thanks, good catch.